packages/components/psammead-image-placeholder/src/index.amp.jsx

```diff
@@ -0,0 +1,91 @@
+/* eslint react/prop-types: 0 */
+import React from 'react';
+import { bool } from 'prop-types';
+import { C_LUNAR, C_SHADOW } from '@bbc/psammead-styles/colours';
+import { BBC_BLOCKS, BBC_BLOCKS_DARK_MODE } from '@bbc/psammead-assets/svgs';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+const bgImageDark = `data:image/svg+xml;base64,${BBC_BLOCKS_DARK_MODE}`;
+const bgImageRegular = `data:image/svg+xml;base64,${BBC_BLOCKS}`;
+
+const AmpImgPlaceholderContainer = ({
+  darkMode,
+  fallback,
+  placeholder,
+  children,
+}) => {
+  return (
+    <div
+      style={{ backgroundColor: `${darkMode ? C_SHADOW : C_LUNAR}` }}
+      fallback={fallback}
+      placeholder={placeholder}
+    >
+      {children}
+    </div>
+  );
+};
+
+const AmpImgPlaceholder = props => (
```

Comment:
Attempted to also use emotion here but for some reason the styling wasn't applying. Here's the code I tried: 

```jsx
const AmpImgPlaceholder = styled('amp-img')`
  position: 'absolute';
  top: '50%';
  left: '50%';
  transform: 'translate(-50%, -50%)';
`;
```

Reply:
Hmm, maybe worth raising an issue in the emotion repo, there seems to be patchy support for styling amp components?

Reply:
We're currently on version `11.0.0` https://github.com/bbc/psammead/blob/4737f88e2cf5efab353b32c916817c2615d23c23/package.json#L46-L48

There was a new version `11.1.5` that came out a few days ago which included the fix from https://github.com/emotion-js/emotion/issues/2224.

But it looks like it's been reverted https://www.npmjs.com/package/emotion, probably due to the Opera mini mishap. So currently there isn't a live build with the fix. Although I tested it before it was reverted and it worked a treat👌